README.md

```diff
@@ -9,6 +9,33 @@ for production use.**
 **One place for service teams to find styles, components and patterns for
 designing government services.**
 
+## GOV.UK Frontend packages
+
+Design System consumes [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend) packages via [NPM](https://www.npmjs.com/).
+These are defined in the [package.json](package.json) file.
+
+---------------------
+**NOTE:**
+For the time being we are consuming private packages. To access private packages, you will first need to log in to NPM with
+
+`npm login`
+
+Credidentials are stored in the  [Design System team credential](https://github.com/alphagov/design-system-team-credentials) repo in `npm/govuk-patterns-and-tools` section. You need SC clearance to access these.
```

Comment:
I don't think it's useful to go into access levels here – if we changed our policy this is the sort of documentation that we'd forget to update.